Formula/bcoin.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class Bcoin < Formula
+  desc "Javascript bitcoin library for node.js and browsers"
+  homepage "https://bcoin.io"
+  url "https://github.com/bcoin-org/bcoin/archive/v2.0.0.tar.gz"
+  sha256 "ff99b735d3b23624455d72eada0de7f7ba4207f4a4cc0cab963ca2036de254a3"
+  depends_on "python" => :build
```

Comment:
Please try with
```suggestion
  depends_on "python@3.8" => :build
```

Reply:
Updated to use `python@3.8`, I originally thought that `node-gyp` required python 2.7 but looks like it works fine with 3.8 according to the `node-gyp` docs.